src/rgw/rgw_common.cc

```diff
@@ -1991,6 +1991,7 @@ bool RGWUserCaps::is_valid_cap_type(const string& tp)
                                     "users",
                                     "buckets",
                                     "metadata",
+				    "info",
```

Comment:
Wrong indent? Or is it Github rendering confusing?

Reply:
will check